Arrangement_on_surface_2/include/CGAL/Arr_dcel_base.h

```diff
@@ -762,10 +765,18 @@ class Arr_inner_ccb : public In_place_list_base<Arr_inner_ccb<V,H,F> >
   typedef Arr_face<V,H,F>                    Face;
   typedef typename Face::Inner_ccb_iterator  Inner_ccb_iterator;
 
+#ifndef CGAL_ARRANGEMENT_DISABLE_UNION_SET_CCB
+  typedef CGAL::Union_find<Self*>            Uf;
+  typedef typename Uf::handle                Uf_handle;
```

Comment:
you probably want to have a less generic name for the typedefs and variables

Reply:
Such as?